service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/utils/CodeableConceptMappingUtils.java

```diff
@@ -14,4 +16,11 @@
         }
     }
 
+    public static boolean hasCode(CodeableConcept code, List<String> codeLists) {
+        return code != null && code.getCoding()
```

Comment:
[optional] or use `Optional.ofNullable`

Reply:
the outcome of this looks like
`return Optional.ofNullable(code).stream()
        .map(CodeableConcept::getCoding)
        .flatMap(Collection::stream)
        .map(Coding::getCode)
        .anyMatch(codeLists::contains);`

Will leave as is